src/app/helpers/error.interceptor.spec.ts

```diff
@@ -1,16 +1,24 @@
 import {TestBed} from '@angular/core/testing';
 import {HttpClientTestingModule} from '@angular/common/http/testing';
 import {ErrorInterceptor} from './error.interceptor';
-import {httpHandlerSpy, httpRequestSpy} from './spies';
+import {httpHandlerSpy, httpRequestSpy, localStorageSpy} from './spies';
 import {HttpHandler, HttpRequest} from '@angular/common/http';
 import {throwError} from 'rxjs';
 
+class MockRouter {
```

Comment:
I was thinking we could use routerSpy in the `spies.ts` file instead of having a new class here. It has also mocked the `navigate` method

Reply:
Yes, there are a number of places this has been used, I'm done rectifying all of them